crates/vm/src/rel_ops.rs

```diff
@@ -12,7 +12,9 @@ pub(crate) trait ResultExt<T> {
 /// A trait for dealing with fallible iterators for the database.
 pub trait RelOps<'a> {
     fn head(&self) -> &Arc<Header>;
-    fn row_count(&self) -> RowCount;
+    fn row_count(&self) -> RowCount {
```

Comment:
Should we remove it entirely or is it being used somewhere else?

Reply:
Is used in `RelOps::collect_vec` and other few places. Only beneficial for projections and queries without predicates.